read/store.go

```diff
@@ -0,0 +1,5 @@
+package read
+
+type Store interface {
```

Comment:
Not sure I follow understanding this new interface approach so would like to talk it through a little

Reply:
It's not new: I was doing what I could to keep consistent with the other dude's style/structure.